en/controllers/middleware.rst

```diff
@@ -68,15 +68,16 @@ process, you can use the ``MiddlewareQueue`` object to attach middleware::
     namespace App;
 
     use Cake\Http\BaseApplication;
+    use Cake\Http\MiddlewareQueue;
     use Cake\Error\Middleware\ErrorHandlerMiddleware;
 
     class Application extends BaseApplication
     {
-        public function middleware($middlewareQueue)
+        public function middleware(MiddlewareQueue $middleware): MiddlewareQueue
```

Comment:
Cant we keep the Queue prefix for clarity?
$middleware->add(Middleware) sounds wrong to me.

Reply:
Sure. I was trying to pare down the line lengths.